### DISCUSSION TITLE
[Curso] - RocketSeat.

### DISCUSSION THREAD

**Discussion by ar1th2ur3** [frontendbr/forum#2295] [Discussão]
Bom dia / noite.

Uma dúvida. 

Estou iniciando na **RocketSeat** mas já possuo uma breve ( bem simples ) bagagem em programação, como HTML, CSS, JavaScript, e C. 

Gostaria de saber se mais alguém também já fez / está fazendo o Curso.

Valeu.

*Não sei se é desse jeito que se deve formular um post off-topic.

**Comment by felquis** [comment]
Olá, não conheço os cursos dessa escola. Mas por favor, de uma olhada nos outros posts aqui no fórum pois é algo bastante perguntado https://github.com/frontendbr/forum/discussions?discussions_q=rocketseat pode ser que te ajude ok? Abraço.

**Comment by bpires** [comment]
Opa @ar1th2ur3 não sei de qual curso específico você está falando, mas imagino que fosse o Explorer. Estou fazendo o curso, um pouco mais da metade dele e tem sido bem proveitoso. Se sua bagagem é breve, ele é uma boa opção para você. Na verdade, é interessante que você não parta do zero (apesar deles falarem que é para quem começa do zero). 

Em relação a conteúdo ele é satisfatório, mas você pode encontrar cursos mais acessíveis se o preço for salgado (como por exemplo o ProgramadorBR, que também é fullstack e sai por um quarto do preço; ou o Origamid (frontend) que o acesso anual sai ainda mais barato). O diferencial dele mesmo é ser bem voltado à prática (learn-by-doing) e focar também em softskills e ter uma comunidade bem integrada. De toda forma, ele dá uma apanhado geral e abrangente do front e do backend, depois é você seguir aprofundando em outros cursos.

Acho um bom ponto de partida, mas te sugiro analisar teu orçamento, ver várias alternativas, testar, abusar dos "trials", comprar, passar os 7 ou 15 dias que derem para pedir estorno explorando ao máximo para tomar uma decisão consciente. São muitas opções e toda hora aparece um novo.. agora também, depois de escolher um, vai e completa o ciclo e faz todo antes de partir para um próximo! 😁  Abs!
